endpoints-framework/src/main/java/com/google/api/server/spi/config/Description.java

```diff
@@ -23,7 +23,7 @@
 /**
  * Annotation to specify the description of an API parameter.
```

Comment:
The Javadoc should be updated to explain it can be used on Enum values.
Also, as it can be used on any Field, the Javadoc should mention very clearly is should not be used on resource fields (use @ApiResourceProperty instead).

Reply:
Ok for enum constants.
But I can't tell to use @ApiResourceProperty since the changes are in another PR.